docs/custom-filters.adoc

```diff
@@ -366,7 +366,7 @@ currently do not have a mechanism for your Filter to know the supported API vers
 you use a hardcoded API version as shown, your Filter will only work with upstream brokers that support that api version
 for that RPC (see issue {github}/issues/438[#438]).
 
-The out-of-band Request/Response are eligible to be intercepted by Filters upstream of this Filter.
+The out-of-band Request/Response will be intercepted by Filters upstream of this Filter.
```

Comment:
I think this covers my thoughts. `are eligible to` made sense in light of the issues identified `will` clearly states the result of the fix 😁